importers/imports.go

```diff
@@ -308,6 +308,24 @@ func NewDefaultImports() Collection {
 				`_ "github.com/denisenkom/go-mssqldb"`,
 			},
 		},
+		"crdb": {
+			Standard: List{
+				`"bytes"`,
+				`"database/sql"`,
+				`"fmt"`,
+				`"io"`,
+				`"os"`,
+				`"os/exec"`,
+				`"strings"`,
+			},
+			ThirdParty: List{
+				`"github.com/pkg/errors"`,
+				`"github.com/spf13/viper"`,
+				`"github.com/volatiletech/sqlboiler/drivers/sqlboiler-crdb/driver"`,
```

Comment:
We may have to figure out a way around this when we go to separate drivers out. sqlite3 would have the same issue.